decidim-initiatives/spec/system/admin/export_initiative_signatures_spec.rb

```diff
@@ -26,34 +26,4 @@
 
     expect(File.basename(download_path)).to include("votes_#{initiative.id}.pdf")
   end
-
-  describe "`collect_user_extra_fields` setting" do
```

Comment:
Same here, we should add a spec (if it's possible)

Reply:
I am not 100% sure how to add it. As the PDF is kind of closed. maybe something like : expect objec to receive something from that column exactly x times.

Reply:
I have tried this, but i could not do it... I have not managed to intercept the `layout.text` method so that i could test what flows through it.